Sources/SQLKit/SQLRowDecoder.swift

```diff
@@ -1,8 +1,28 @@
-struct SQLRowDecoder {
-    func decode<T>(_ type: T.Type, from row: SQLRow, prefix: String? = nil) throws -> T
+public struct SQLRowDecoder {
+    public var keyPrefix: String? = nil
+    public var keyDecodingStrategy: KeyDecodingStrategy = .useDefaultKeys
```

Comment:
Vapor style is to create an explicit `init` method for these (even if `internal`).